components/brave_ads/browser/ads_service.h

```diff
@@ -32,6 +32,10 @@ class AdsService : public KeyedService {
   virtual void OnMediaStart(SessionID tab_id) = 0;
   virtual void OnMediaStop(SessionID tab_id) = 0;
   virtual void ClassifyPage(const std::string& url, const std::string& page) = 0;
+  virtual bool HasEnabledAds() const = 0;
```

Comment:
I think this naming is a bit confusing if you don't know the purpose of it. Seems like all we need is an `IsFirstLaunch` method?

Reply:
Per our discussion in slack, for needing to know whether or not this notification should be shown, we need to know if they have ever enabled Ads at all. If we were checking `IsFirstLaunch` when firing up Ads at that point the notification would be redundant.